lib/web_ui/lib/src/engine/compositor/initialization.dart

```diff
@@ -9,7 +9,7 @@ const bool experimentalUseSkia =
     bool.fromEnvironment('FLUTTER_WEB_USE_SKIA', defaultValue: false);
 
 /// The URL to use when downloading the CanvasKit script and associated wasm.
-const String canvasKitBaseUrl = 'https://unpkg.com/canvaskit-wasm@0.7.0/bin/';
+const String canvasKitBaseUrl = 'https://particles.skia.org/static/';
```

Comment:
Will this make our tests unstable? Maybe host our own copy on a firebase static site? This way we can control when to upload a new version.

Reply:
There are currently no CanvasKit tests. We can worry about this when we enable testing for CanvasKit

Reply:
Ah right, one more thing to add to our testing strategy :)